### PR TITLE
Bridge hot-update is supported.

### PR DESCRIPTION
**Pull request by wanghaEMQ**
* Hot-updating bridge is supported.
* Adjust the name of interfaces in bridge.c and make them readable.
* Add some fields in conf_bridge_node to implement reload bridge node.
* Add a comment to web_server.c about a process with low performance.
* Reduce one shot of calculating in length.

### CODE REVIEW

**Comment by JaylinYu** [comment]
Plz double check! it doesnt work!

**Review by JaylinYu — CHANGES_REQUESTED** [review]
plz refer to my previous PR

File: nanomq/bridge.c [hunk 1]
```diff
@@ -891,8 +891,7 @@ bridge_client(nng_socket *sock, conf *config, conf_bridge_node *node)
 	node->bridge_aio = nng_alloc(
 	    (config->parallel + node->parallel * 2) * sizeof(nng_aio *));
 
-	for (uint32_t num = 0; num < (config->parallel + node->parallel * 2);
-	     num++) {
+	for (uint32_t num = 0; num < config->parallel; num++) {
```
[the remark this answers is not in the document]
Comment (wanghaEMQ) [reply nanomq/bridge.c]: Sorry. The change was removed.

**Review by JaylinYu — APPROVED** [review]
This is a good start

**Review by Hermann0222 — APPROVED** [review]

**Review by Hermann0222 — APPROVED** [review]

File: nanomq/web_server.c [hunk 2]
```diff
@@ -403,6 +403,7 @@ inproc_cb(void *arg)
 
 		// response to client
 		nng_msg_alloc(&work->msg, sizeof(http_msg));
+		// TODO performace bottlenect here (Copy structure 3 times)
```
Comment (JaylinYu) [inline nanomq/web_server.c:406]: nevermind, webserver is not meant to be performant

**Review by JaylinYu — COMMENTED** [review]
I have to revert this first

**Pull request merged**